src/main/features/response/views/evidence.njk

```diff
@@ -1,6 +1,10 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import csrfProtection, linkButton, textArea, saveAndContinueButton, errorSummary %}
```

Comment:
you removed textArea macro further down but not here?

Reply:
done